src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -1458,6 +1470,16 @@ BuildSubqueryJobQuery(MultiNode *multiNode)
 	/* build the where clause list using select predicates */
 	whereClauseList = QuerySelectClauseList(multiNode);
 
+	hasAggregates = contain_agg_clause((Node *) targetList) ||
```

Comment:
I think our coding conventions is more like;
```c
if (contain_agg_clause((Node *) targetList) || contain_agg_clause((Node *) havingQual))
{
    hasAggregates = true;
}
```

Reply:
I was only copying code from below and setting it to a local variable. 
made the suggested change